include/openmc/position.h

```diff
@@ -63,6 +66,20 @@ inline Position operator*(Position a, Position b) { return a *= b; }
 inline Position operator*(Position a, double b)   { return a *= b; }
 inline Position operator*(double a, Position b)   { return b *= a; }
 
+inline bool operator==(Position a, Position b)
+{return a.x == b.x && a.y == b.y && a.z == b.z;}
+inline bool operator==(Position a, double b)
```

Comment:
At the time I made this I was thinking that it only made since for comparisons with the origin, e.g. `if (r == 0)`, but you're right that it's not intuitive so I'm now switching to `if (r == Position(0, 0, 0))`